src/link/script.y

```diff
@@ -411,6 +469,16 @@ static void alignTo(uint32_t alignment, uint32_t alignOfs) {
 
 static void pad(uint32_t length) {
 	auto const &context = lexerStack.back();
+	if (activeType == SECTTYPE_INVALID) {
+		scriptError(context, "Cannot modify PC because no memory region is active");
+		return;
+	}
+
+	if (isPcFloating) {
+		floatingAlignOffset = (floatingAlignOffset + length) % floatingAlignMask;
```

Comment:
Noting where the `% floatingAlignMask` happens, which makes sense iff it's really a "`floatingAlignSize`" value like `1 << N` not `(1 << N) - 1`.